chains/evm/cli/bridge/vote-proposal.go

```diff
@@ -0,0 +1,105 @@
+package bridge
+
+import (
+	"fmt"
+
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls/contracts/bridge"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls/evmtransaction"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls/transactor"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/cli/flags"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/cli/initialize"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/cli/logger"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/voter/proposal"
+	"github.com/ChainSafe/chainbridge-core/util"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+var voteProposalCmd = &cobra.Command{
+	Use:   "vote-proposal",
+	Short: "Vote on proposal",
+	Long:  "Votes on an on-chain proposal. Valid relayer private key required for transaction to be successful.",
```

Comment:
This is not following our new CLI naming guideline (only place where you can see it, for now, is [this closed PR](https://github.com/ChainSafe/chainbridge-core/pull/196)), but I guess this is something we should add somewhere - maybe even inside README file? @waymobetta @P1sar what do you think?

Reply:
To be more precise, only this `Long` description is not following guidelines.

Reply:
Yeah, will update. Wrote this before the guideline.